src/puppetlabs/ezbake/core.clj

```diff
@@ -808,7 +829,8 @@ Additional uberjar dependencies:
                              #(deputils/expand-snapshot-versions
                                 lein-project % {:reproducible? reproducible?}))]
     (let [template-dir (get-template-file build-target)
-          uberjar-name (:uberjar-name lein-project)]
+          uberjar-name (get lein-project :uberjar-name
+                            (format "%s-%s-standalone.jar" (:name lein-project) (:version lein-project)))]
```

Comment:
How will this affect existing `uberjar-name`s? It's been a while since I Clojure'd, so I wanna make sure we know what effects a name change might have.

Reply:
![](https://media.giphy.com/media/E6LPrrzKOJwhG/giphy.gif)